packages/core/src/state-management/repository.ts

```diff
@@ -107,14 +107,15 @@ export class Repository {
     }
   }
 
-  private async fromNetwork(streamId: StreamID, opts: LoadOpts): Promise<RunningState> {
+  private async fromNetwork(streamId: StreamID): Promise<RunningState> {
     const handler = this.#deps.handlers.get(streamId.typeName);
     const genesisCid = streamId.cid;
     const commit = await this.#deps.dispatcher.retrieveCommit(genesisCid, streamId);
     if (commit == null) {
       throw new Error(`No genesis commit found with CID ${genesisCid.toString()}`);
     }
-    const state = await handler.applyCommit(commit, streamId.cid, this.#deps.context);
+    // Do not check timestamp here
+    const state = await handler.applyCommit(commit, {cid: streamId.cid, timestamp: null}, this.#deps.context);
```

Comment:
Note that we don't really want to disable signature verification here. Just disable verifying whether or not the key used to sign has been revoked.

Reply:
Well, it would be safe to completely skip signature verification here so long as we definitely do it later. And I'm not sure it will be straightforward to disable the key revocation check without skipping all of signature verification.

Reply:
Actually, it is simpler then it sounds. `timestamp = 1` makes the trick. See https://github.com/ceramicnetwork/js-ceramic/pull/1518

Reply:
Just made a comment there - I'm not sure that actually works